pre_commit_all_the_way_down/python_doc.py

```diff
@@ -33,29 +34,52 @@
 TRAILING_NL_RE = re.compile(r"\n+\Z", re.MULTILINE)
 
 
+@dataclass
+class Context:
+    filename: str
+    start: int = None
+    end: int = None
```

Comment:
otherwise this won't type check
```suggestion
    start: Optional[int] = None
    end: Optional[int] = None
```